IncrementSlider/utils.js

```diff
@@ -16,24 +15,7 @@ const calcStep = (knobStep, step) => {
 	// otherwise, increment/decrement would be no-ops
 	return s || 1;
 };
-
-const isIncrement = ({keyCode}, {orientation}) => {
-	return orientation === 'vertical' ? is('up', keyCode) : is('right', keyCode);
-};
-
-const isDecrement = ({keyCode}, {orientation}) => {
-	return orientation === 'vertical' ? is('down', keyCode) : is('left', keyCode);
-};
-
-const isNotMax = (ev, {value, max}) => {
-	return value !== max;
-};
-
-const isNotMin = (ev, {min, value = min}) => {
-	return value !== min;
-};
-
-const emitChange = (direction) =>  adaptEvent(
+export const emitChange = (direction) =>  adaptEvent(
```

Comment:
```suggestion

export const emitChange = (direction) =>  adaptEvent(
```
Please add a new line. Other changes look good to me.